var/spack/repos/builtin/packages/steps/package.py

```diff
@@ -0,0 +1,78 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Steps(CMakePackage):
+    """STochastic Engine for Pathway Simulation"""
+
+    homepage = "https://groups.oist.jp/cnu/software"
+    git      = "https://github.com/CNS-OIST/STEPS.git"
+
+    version("3.3.0", submodules=True)
+    version("3.2.0", submodules=True)
+    version("develop", branch="master", submodules=True)
+
+    variant("native", default=True, description="Generate non-portable arch-specific code")
+    variant("lapack", default=False, description="Use new BDSystem/Lapack code for E-Field solver")
+    variant("petsc", default=False, description="Use PETSc library for parallel E-Field solver")
+    variant("mpi", default=True, description="Use MPI for parallel solvers")
+
+    depends_on("blas")
+    depends_on("lapack", when="+lapack")
+    depends_on("mpi", when="+mpi")
+    depends_on("petsc^debug+int64", when="+petsc")
```

Comment:
Do we have a package called `debug`? Or was this supposed to be a variant of `petsc`?

Reply:
Hello @alalazo 
This is supposed to be a variant of `petsc`. Should I use `~` instead of `^`?

Reply:
@alalazo Could you please provide feeback?

Reply:
The character `^` is used for dependencies. You should use `~` if you want to deactivate a variant and `+` if you want to activate it.

Reply:
I fixed the spec to properly deactivate the `debug` variant.